bedrock/admin/templates/admin/base_site.html

```diff
@@ -1,5 +1,5 @@
 {% extends "admin/base.html" %}
-
+{% comment %} SKIP LICENSE INSERTION {% endcomment %}
```

Comment:
what's this for?

Reply:
To stop pre commit hooks adding a jinja formatted comment when there's already a Django formatted comment.  Was the easiest fix for a niche problem